packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props

```diff
@@ -161,4 +161,11 @@
     <SupportedTargetFramework Remove="@(_UnsupportedNETCoreAppTargetFramework);@(_UnsupportedNETStandardTargetFramework);@(_UnsupportedNETFrameworkTargetFramework)" />
   </ItemGroup>
 
+  <!--
+    Import Windows Forms props.
+    These come via the Windows Forms transport package, that can be found under
+    https://github.com/dotnet/winforms/tree/main/pkg/Microsoft.Private.Winforms/sdk
+  -->
+  <Import Project="System.Windows.Forms.props" />
```

Comment:
It might be too late (_or not_) How about `Microsoft.NET.Desktop.WindowsForms` for WinForms specific props/targets and `Microsoft.NET.Desktop.WPF` for WPF specific props/targets?

We can also have `Microsoft.NET.WindowsDesktop.WindowsForms` or `Microsoft.NET.WindowsDesktop.Forms` and `Microsoft.NET.WindowsDesktop.WPF`, similar naming to `Microsoft.NET.WindowsDesktop.App.<WindowsForms|WPF>` packages.

Reply:
We're having an offline discussion with @ryalanms and @SamBent about this.

Reply:
Based on the discussion in the WinForms/WPF teams meeting, WinForms has additional upcoming work that will require other WinForms-specific props/targets.  This is only the first of several changes.  Taking on the additional complexity of moving files in the WinForms transport package to be included in the WindowsDesktop SDK (in the WPF repo) seems reasonable; these \*WinForms\*.props/targets will only be imported when UseWinForms is true.   WPF will also scope our files to WPF-only.

Reply:
Condition="'$(UseWindowsForms)' == 'true'"

Reply:
No, we can't conditionally import as `UseWindowsForms` isn't defined yet
![image](https://user-images.githubusercontent.com/4403806/126591657-37230ccc-11b4-4056-92e6-eab7b3a020b8.png)